docs/usage.md

```diff
@@ -62,6 +62,48 @@ the cloned repository.
 evg-module-manager disable --module wiredtiger
 ```
 
+## Creating a branch
+
+Use `create-branch` command to create a new branch on base repo and any enabled modules.
+To create a new branch, you can use the revision you want to work with `-r` or `--revsion` option.
```

Comment:
It seems like the most common use case for `create-branch` is going to be creating a branch from HEAD. With that in mind, I wonder if it would make sense to reorder this section a little. 

* Start off with a with a brief description of `create-branch`, which you already have.
* Talk about how to create a branch with a name.
* Give an example of creating a branch with a name.
* Talk about how a revision could also be provided to create a branch based off that revision.
* Give an example of creating a branch with a name and revision.